addons/studio-invite-button/userscript.js

```diff
@@ -0,0 +1,54 @@
+export default async function ({ addon, global, console }) {
+// RegExp to test for a valid url and get the studio id
+const urlRegex = /^https?:\/\/scratch\.mit\.edu\/studios\/(\d+)\/comments($|\/)/;
+// invites a user to the studio
+function inviteToStudio(user) {
+  return fetch(`https://scratch.mit.edu/site-api/users/curators-in/${urlRegex.exec(window.location.href)[1]}/invite_curator/?usernames=${user}`, {
+    headers: {
+      'x-csrftoken': addon.auth.csrfToken,
+      'x-requested-with': 'XMLHttpRequest',
+      referer: 'https://scratch.mit.edu',
+
+    },
+    body: null,
+    method: 'PUT',
+  });
+}
+// checks if the user is a manager and the URL is correct
+if (document.getElementById('description').classList.contains('editable') && urlRegex.test(window.location.href)) {
+  Array.from(document.getElementsByClassName('comment')).forEach((comment) => {
+    const inviteButton = document.createElement('span'); // create the button
+    inviteButton.innerHTML = 'Invite';
```

Comment:
Wherever you set the innerHTML to just text, change it to be innerText.

Reply:
Also, use l10n

Reply:
> Wherever you set the innerHTML to just text, change it to be innerText.

thanks